src/main/java/org/mockito/Mockito.java

```diff
@@ -2702,4 +2696,17 @@ static MockitoDebugger debug() {
     public static MockitoFramework framework() {
         return new DefaultMockitoFramework();
     }
+
+    /**
+     * Allows to manage Mockito strictness (debugging, cleaner tests) without JUnit.
+     * <p>
+     * TODO javadoc
+     *
+     * @param testClassInstance
+     * @param strictness
+     */
+    @Incubating
+    public static MockitoMocking startMocking(Object testClassInstance, Strictness strictness) {
+        return new DefaultMockitoMocking(testClassInstance, strictness, new ConsoleMockitoLogger());
+    }
```

Comment:
* While public API is ok for me, I wonder if this API should be exposed in the Mockito class at first. Why not a class like `MockitoMocking.startMocking()...`, and rename the current `MockitoMocking` interface to `OngoingMockitoMocking`.
* This method maybe more interesting if started some kind of builder instead of taking arguments. One could want to modify the logger. And future options could be passed much more easily without breaking binary compatibility.